rakelib/artifacts.rake

```diff
@@ -90,15 +156,15 @@ namespace "artifact" do
     path = matches.first[gem_line_regex, 1]
 
     if path
-      Rake::Task["plugin:install-local-core-gem"].invoke("logstash-core", path)
+      Rake::Task["plugin:build-local-core-gem"].invoke("logstash-core", path)
     else
-      puts("[artifact:install-logstash-core] using logstash-core from Rubygems")
+      exit(1)
```

Comment:
Why are we explicitly returning a value greather than 0 ?
I don't see any error in the code path that would require us to return 1 ?